src/java.base/share/classes/java/lang/ref/PhantomReference.java

```diff
@@ -91,7 +91,7 @@ boolean refersToImpl(T obj) {
      * @param referent the object the new phantom reference will refer to
      * @param q the queue with which the reference is to be registered,
      *          or {@code null} if registration is not required
-     * @throws IllegalArgumentException if the referent is an instance of an
+     * @throws IdentityException if the referent is an instance of an
```

Comment:
Instead this should refer to isIdentity(), the implementation uses Objects.isIdentityObject(referent).
Perhaps:
@throws IdentityException if the referent is not an {@link Objects#isIdentityObject(Object o) identity object}.

The same applied to SoftReference and WeakReference. (below)

Reply:
updated.